### PR TITLE
Add SCC to support Openshift

### PR DESCRIPTION
**Pull request by airadier**
Similar to https://github.com/falcosecurity/charts/pull/51, add SCC support for Openshift, required to mounth the grpc socket as hostPath

Signed-off-by: Alvaro Iradier <airadier@gmail.com>

**What type of PR is this?**

/kind chart-release

/area falco-exporter-chart

**What this PR does / why we need it**:

The daemonset fails to create the pods in Openshift environment due to the request of HostPath mount, required to connect to Falco via GRPC.

This PR adds a SCC resource which is only created in Openshift (detected via API capabilities in the chart), and that can be disabled with scc.create: false in the values.yaml

**Checklist**
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] CHANGELOG.md updated

### CODE REVIEW

**Review by leogr — COMMENTED** [review]
Thank you @airadier !

It seems good to me, I have just to find the time to try it.

/cc @nibalizer

**Comment by airadier** [comment]
> Thank you @airadier !
> 
> It seems good to me, I have just to find the time to try it.
> 
> /cc @nibalizer

No problem @leogr, I just tested on an Openshift cluster with and without the SCC option. If you don't want to try yourself I can just show you.

Regards.

**Comment by leogr** [comment]
/hold

This PR needs to be rebased

**Comment by leogr** [comment]
Hey @airadier 

Could you please rebase this PR on top of the current master branch and bump the version again?
We need that to merge the PR and trigger a new release.

I will be happy to approve this PR again once that has been fixed.

Thank you in advance

**Review by leogr — APPROVED** [review]
Thank you!

/hold cancel

**Pull request merged**
